Integrations/Tanium_v2/Tanium_v2.py

```diff
@@ -108,7 +108,38 @@ def parse_question(self, text, parameters):
             except Exception:
                 raise ValueError('Failed to parse question parameters.')
 
-        res = self.do_request('POST', 'parse_question', {'text': text}).get('data')[0]
+            res = self.do_request('POST', 'parse_question', {'text': text}).get('data')[0]
+        else:
+            # if there are no parameters argument - try to gets the sensors from parse question api
+            text_without_params = re.sub(r'\[(.*?)\]', '', text)
+            res = self.do_request('POST', 'parse_question', {'text': text_without_params}).get('data')[0]
+
+            # call sensors/by-name/ for each sensor in the response and update parameters_condition
+            # with the correct parameters
+            for item in res.get('selects'):
+                sensor = item.get('sensor').get('name')
+                search_results = re.search(rf'{sensor}\[(.*?)\]', text)
+                if search_results:
+                    parameters_str = search_results.group(1)
+                    parameters = parameters_str.split(',')
+                    endpoint_url = f'sensors/by-name/{sensor}'
+                    sensor_response = self.do_request('GET', endpoint_url)
+                    sensor_response = self.get_sensor_item(sensor_response.get('data'))
+
+                    tmp_item = {'sensor': sensor, 'parameters': []}
+                    index = 0
+                    for param in parameters:
+                        if param:
+                            if param == '""':
+                                param = ''
+                            key = sensor_response['Parameters'][index]['Key']
+                            tmp_item['parameters'].append({
+                                'key': '||' + key + '||',
+                                'value': param
+                            })
+
+                        index = index + 1
+                    parameters_condition.append(tmp_item)
```

Comment:
original looked like C...

```suggestion
                    for param, sensor_key in zip(parameters, sensor_response['Parameters']):
                        if param:
                            if param == '""':
                                param = ''
                            tmp_item['parameters'].append({
                                'key': '||' + sensor_key['Key'] + '||',
                                'value': param
                            })

                    parameters_condition.append(tmp_item)
```